advocacy_docs/partner_docs/QuestFoglightforDatabases/05-UsingQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Using Quest Foglight for Databases to Monitor Health of EDB Postgres Advanced Server and/or EDB Postgres Extended Server Databases'
+description: 'Walkthroughs of Quest Foglight database monitoring'
```

Comment:
The description should be kept standard like the rest of the guides, lets change it to "Walkthroughs of multiple Quest Foglight for Databases usage scenarios"